app/src/renderer/js/main.js

```diff
@@ -220,104 +195,130 @@ document.addEventListener('DOMContentLoaded', () => {
         temporary: true,
         forHowLong: 1000
       });
-      shake(this);
+      shake(event.currentTarget);
       ipcRenderer.send('open-editor-window', {notify: true});
     } else {
       prepareRecordButton();
     }
   };
 
+  const setSelectedRatio = (width, height) => {
+    dimensions.ratio = findRatioForSize(width, height);
+
+    // Remove app from dimensions object
+    // since size is being set manually
+    if (dimensions.app && (dimensions.app.width !== width || dimensions.app.height !== height)) {
+      dimensions.app = null;
+    }
+
+    dimensionsEmitter.emit('change', dimensions);
+    app.kap.settings.set('dimensions', dimensions);
+  };
+
+  // Set initial values
+  inputWidth.value = Math.round(width);
+  inputHeight.value = Math.round(height);
+
+  setSelectedRatio(width, height);
+
+  if (ratioLocked === true) {
+    linkBtn.classList.toggle('is-active');
+  }
+
+  handleTrafficLightsClicks({hide: true});
+
   recordBtn.addEventListener('click', handleRecord);
   ipcRenderer.on('record', handleRecord);
   ipcRenderer.on('crop', handleRecord);
 
-  function setCropperWindowSize(width, height) {
+  const setCropperWindowSize = (width, height) => {
     ipcRenderer.send('set-cropper-window-size', {
       width: width || lastValidInputWidth,
       height: height || lastValidInputHeight
     });
-  }
+  };
 
-  function handleWidthInput(event, validate) {
+  const handleWidthInput = (event, validate) => {
     // User is deleting the current value
     // to enter a new one
-    if (!this.value) {
+    if (!event.target.value) {
```

Comment:
Had to keep `target` here because somewhere `currentTarget` is set to `null`. Probably because we are emitting this event manually in `handleKeyDown`.